### PR TITLE
Events tweaks

### PR DESCRIPTION
**Pull request by bmuenzenmeyer**
## Description

@ovflowd asked for some tweaks to the upcoming events page based on TSC feedback. 

After some review, this page is far simpler.

image

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [x] I have run `npx turbo test` to check if all tests are passing.
- [x] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: components/MDX/Calendar/RecentEventPosts.tsx [hunk 1]
```diff
@@ -6,20 +6,17 @@ import getBlogData from '@/next-data/blogData';
 
 import styles from './calendar.module.css';
 
-const UpcomingSummits: FC = async () => {
+const RecentEventPosts: FC = async () => {
```
Comment (bmuenzenmeyer) [inline components/MDX/Calendar/RecentEventPosts.tsx]: this was never only pulling summit data, so renamed throughout

File: components/MDX/Calendar/UpcomingMeetings.tsx [hunk 2]
```diff
@@ -9,9 +9,9 @@ import type { CalendarEvent } from '@/types';
 
 import styles from './calendar.module.css';
 
-type GrouppedEntries = Record<string, Array<CalendarEvent>>;
+type GroupedEntries = Record<string, Array<CalendarEvent>>;
```
Comment (bmuenzenmeyer) [inline components/MDX/Calendar/UpcomingMeetings.tsx:12]: typo

File: components/MDX/Calendar/UpcomingMeetings.tsx [hunk 3]
```diff
@@ -9,9 +9,9 @@ import type { CalendarEvent } from '@/types';
...
 
-const UpcomingEvents: FC = async () => {
+const UpcomingMeetings: FC = async () => {
```
Comment (bmuenzenmeyer) [inline components/MDX/Calendar/UpcomingMeetings.tsx:14]: events is too loaded a term, as it is really pulling meeting data from the calendar, not `events` data from the blog

File: pages/en/about/get-involved/events.mdx [hunk 4]
```diff
@@ -3,22 +3,26 @@ title: Upcoming Events
...
 
-## Upcoming Node.js® Summits
+## Upcoming Events
 
-Interested in joining a [Collaborator Summit](/about/get-involved/collab-summit) hosted by the Node.js project?
-Check out the list below to find upcoming events.
+Node.js events are open and available to the public. Anyone is welcome to join and participate.
 
-Browse [previous Collaborator Summits & Events](/blog/events/) hosted by Node.js.
+[Collaborator Summits](/about/get-involved/collab-summit) hosted by the Node.js project are larger gatherings usually taking place adjacent to a community conference.
```
Comment (bmuenzenmeyer) [inline pages/en/about/get-involved/events.mdx]: retained this snippet about the collab summit in case someone finds this page first

**Review by bmuenzenmeyer — COMMENTED** [review]
self-review

**Comment by ovflowd** [comment]
@bmuenzenmeyer I think we can remove the "Event Blogs" if we already have a link that goes to the section. It feels repetitive and doesn't add much value tbh

**Comment by ovflowd** [comment]
We can also remove the reference to collaborator summit as we have a dedicated page

**Comment by bmuenzenmeyer** [comment]
> I think we can remove the "Event Blogs" if we already have a link that goes to the section. It feels repetitive and doesn't add much value tbh

will do later tonight. I thought that section was part of the original ask 😄

**Comment by bmuenzenmeyer** [comment]
@ovflowd does this look better? If so, I'll push. Some code will be obsolete with it

image

**Comment by ovflowd** [comment]
Yea!

**Comment by ovflowd** [comment]
> > I think we can remove the "Event Blogs" if we already have a link that goes to the section. It feels repetitive and doesn't add much value tbh
> 
> will do later tonight. I thought that section was part of the original ask 😄

Right, but then I wonder how relevant it is. I guess we can keep it

**Comment by ovflowd** [comment]
FYI, this branch needs to be rebased

**Comment by bmuenzenmeyer** [comment]
> Right, but then I wonder how relevant it is. I guess we can keep it

removed per agreed upon screenshot.

**Review by ovflowd — APPROVED** [review]

**Pull request merged**